src/data/analyse_de.json

```diff
@@ -0,0 +1,309 @@
+            },
+            "totalKey": [
+                "qr_teletan_redeemable_cumulated",
+                "qr_redeemable_cumulated",
+                "teletan_redeemable_cumulated"
+            ],
+            "tabs": [
+                {
+                    "label": "Gesamt"
+                },
+                {
+                    "label": "QR-Codes"
+                },
+                {
+                    "label": "TeleTAN"
+                }
+            ] 
+        },
+        {
+            "title": "Warnungen",
+            "export": "https://obs.eu-de.otc.t-systems.com/obs-public-dashboard/csv/cwa_ppa_warnings_data.zip",
+            "info": {
+                "content": "<strong>Täglich</strong><p>Tagesgenaue Anzahl roter und grüner Risikowarnungen sowie deren Summe in der Corona-Warn-App.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl roter und grüner Risikowarnungen sowie deren Summe in der Corona-Warn-App (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl roter und grüner Risikowarnungen sowie deren Summe in der Corona-Warn-App.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe roter und grüner Risikowarnungen sowie deren Summe in der Corona-Warn-App seit Beginn der Messung.</p><br><strong>Weitere Informationen</strong><p>Eine grüne Risikowarnung erhalten Nutzende, die in den letzten 14 Tagen eine kurze und/oder räumlich entfernte Begegnung mit einer Corona-positiv getesteten Person hatten. Die Begegnung lag somit nicht über dem definierten Schwellenwert. Für die betroffenen Nutzenden besteht kein aktiver Handlungsbedarf.</p><p>Eine rote Risikowarnung erhalten Nutzende, die in den letzten 14 Tagen eine lang andauernde und/oder räumlich nahe Begegnungen mit mindestens einer Corona-positiv getesteten Person hatten. Die Begegnung lag über dem definierten Schwellenwert. Betroffene Nutzende werden aufgefordert, persönliche Kontakte zu reduzieren. Bei Symptomen soll die Hausarztpraxis, der kassenärztliche Bereitschaftsdienst oder das Gesundheitsamt kontaktiert werden. Die Hausarztpraxis oder das Gesundheitsamt entschieden über ein Testangebot.</p>"
+            },
+            "totalKey": ["ppa_total_warnings_cumulated"]
+        }
+    ],
+    "translations":{
+
+        "analyseBoardTitleTitle":{
+            "0": "Downloads",
+            "1": [
+                "Registrierte Tests",
+                "Testergebnisse",
+                "Testergebnisse"
+            ],
+            "2": "Teilungsverhalten",
+            "3": "Warnungen"
+        },
+
+
+
+        "analyseBoardTitleAppend":{
+            "1": "(kumuliert)",
+            "2": "",
```

Comment:
"2" is empty because if the user has selected "Täglich" there is no term in () after e.g. "Downloads", right?
If yes, why is "2" not "1", since the first field is "Täglich" and not "kumuliert" (which is "1" currently)

Reply:
Yes you are right, on the point why it is empty.
The "1" and "2" are in this case an Id (that refers to the switch) and not an sorting order. It is arguably misleading as the ID is numeric.